src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -192,7 +195,7 @@ private char ScanEscapeSequence(out char surrogateCharacter)
             return error;
         }
 
-        private void ScanInterpolatedStringLiteral(bool isVerbatim, ref TokenInfo info)
+        private void ScanInterpolatedStringLiteral(ref TokenInfo info)
```

Comment:
Since `isVerbatim` was removed, the comment below is out-of-date

Reply:
true.  fixed.